lib/helpers.coffee

```diff
@@ -4,6 +4,12 @@ path = require 'path'
 child_process = require 'child_process'
 
 Helpers = module.exports =
+  shouldTriggerLinter: (linter, wasTriggeredOnChange, scopes)->
+    # Trigger fly linters on save, but not save linters on fly
```

Comment:
If you're going to separate these, we need a new category for CoffeeLint which does both.